example/gluon/word_language_model/train.py

```diff
@@ -70,8 +87,19 @@ def batchify(data, batch_size):
 
 
 ntokens = len(corpus.dictionary)
-model = model.RNNModel(args.model, ntokens, args.emsize, args.nhid,
-                       args.nlayers, args.dropout, args.tied)
+assert (not args.vardrop_out and not args.vardrop_state) or args.use_cell, \
+       'variational dropout is only available in cell API.'
+if not args.use_cell:
```

Comment:
Alternative is duplicates. This is the simplest way to include example for RNN cells.